presto-tests/src/test/java/com/facebook/presto/execution/resourceGroups/db/TestQueues.java

```diff
@@ -204,6 +204,35 @@ public void testRejection()
         }
     }
 
+    @Test(timeOut = 240_000)
```

Comment:
Make these tests shorter - we don't want to increase test runtime unless we have to.